template.rb

```diff
@@ -58,9 +57,9 @@ def apply_template!(template_root)
   apply '.template/addons/docker/template.rb'
   apply '.template/addons/heroku/template.rb'
   apply '.template/addons/github/template.rb'
-  apply '.template/addons/semaphore/template.rb'
 
   # Add-ons - [Optional]
+  apply '.template/addons/semaphore/template.rb' if yes?(install_addon_prompt('SemaphoreCI'))
```

Comment:
Can you put a TODO here because later on, we will allow the user can choose only either `Guthub Action or Semaphore`, I think it doesn't make much sense to have both CI service for 1 project (sometimes we have but it's rare)

And with this addon, the user might not be aware that Github Action doesn't include as default so their might choose Semaphore Addon, can we somehow make it clear at this point @longnd 🤔, do you have any idea?

Reply:
@andyduong1920 How about this? - notifying the user about default addons installed
<img width="524" alt="Screen Shot 2021-01-18 at 11 17 50" src="https://user-images.githubusercontent.com/1896814/104871869-cde4b000-597e-11eb-9946-657af94ea8db.png">

I made the change in 01f1f4f.

Reply:
Looking good for me @longnd, but later on, if we add a new `default addon` then we need to modify the `print_default_addons_message` message manually, how about we make that automatically @longnd

Something like 
```
default_addons = {
  docker: docker,
  heroku: heroku,
  github: 'Github along with Github Action'
}
```

Reply:
@longnd if this one is a blocker to merge this PR, then we can `comment the Semaphore addon` now and create an issue to enable Semaphore addon on another PR, what do you think @longnd

Reply:
thanks for your suggestion. Updated in 7953b14.

Reply:
Nice @longnd